R/check-devtools.r

```diff
@@ -133,3 +134,13 @@ check_status <- function(status, name, warning) {
 
   invisible(status)
 }
+
+check_doc_files <- function(pkg) {
+  pkg <- as.package(pkg)
+  doc_path <- file.path(pkg$path, "inst", "doc")
+
+  check_status(length(dir(doc_path)) == 0,
```

Comment:
This should go above `check_status()`, and can you please match the identation style of the `check_match()` call with the other check functions?